src/ONYXKEYS.js

```diff
@@ -49,5 +52,6 @@ export default {
         REPORT_ACTIONS: 'reportActions_',
         REPORT_DRAFT_COMMENT: 'reportDraftComment_',
         REPORT_USER_IS_TYPING: 'reportUserIsTyping_',
+        NVP: 'nvp_',
```

Comment:
I know we talked about this a bit, but seeing it now I think it should not be a "collection". Here's my anti-pitch for doing it:

- everything else so far (report, reportActions, etc) is an object with values that all take the same shape
- they are "collections" in the sense that they are multiples of the same type of thing stored in an object (that acts more like an array)
- it seems like we are using them here as constants. usually whatever comes after the `_` in the collections is going to be some kind of variable like a `reportID`.

Not saying we can't do things this way. But it makes the idea of a "collection" a little more ambiguous and it's already not obvious to begin with.